pkg/test/tpm.go

```diff
@@ -262,15 +262,57 @@ func PSIndexConfig(txtAPI api.ApiInterfaces) (bool, error, error) {
 		}
 		return true, nil, nil
 	case tss.TPMVersion20:
+		raw, err = tpmCon.ReadNVPublic(tpm20OldPSIndex)
+		if err != nil {
+			if !strings.Contains(err.Error(), tpm2NVPublicNotSet) {
+				return false, nil, err
+			}
+		}
 		raw, err = tpmCon.ReadNVPublic(tpm20PSIndex)
 		if err != nil {
+			if strings.Contains(err.Error(), tpm2NVPublicNotSet) {
+				return false, fmt.Errorf("PS indices not set"), err
+			}
 			return false, nil, err
 		}
 		buf := bytes.NewReader(raw)
 		err = binary.Read(buf, binary.BigEndian, &d2.NVIndex)
 		if err != nil {
 			return false, nil, err
 		}
+		err = binary.Read(buf, binary.BigEndian, &d2.NameAlg)
```

Comment:
why? seems not to be covered by commit message?

Reply:
That is also part of missing tpm2 part. I have to read every field manually, because go-tpm implements the structure NVPublic with a slice of bytes with unknown size at a place where an other dynamic structure would be needed. 

At least this works ¯\_(ツ)_/¯

Reply:
Then this should be a separate commit.